test/Interop/Cxx/stdlib/std-module-interface.swift

```diff
@@ -0,0 +1,33 @@
+// RUN: %target-swift-ide-test -print-module -module-to-print=std -source-filename=x -enable-cxx-interop -Xcc -stdlib=libc++ -tools-directory=%llvm_obj_root/bin -module-cache-path %t | %FileCheck %s  -check-prefix=CHECK-STD
+// RUN: %target-swift-ide-test -print-module -module-to-print=std.iosfwd -source-filename=x -enable-cxx-interop -Xcc -stdlib=libc++ -tools-directory=%llvm_obj_root/bin -module-cache-path %t | %FileCheck %s  -check-prefix=CHECK-IOSFWD
+// RUN: %target-swift-ide-test -print-module -module-to-print=std.chrono -source-filename=x -enable-cxx-interop -Xcc -stdlib=libc++ -tools-directory=%llvm_obj_root/bin -module-cache-path %t | %FileCheck %s  -check-prefix=CHECK-CHRONO
+
+// Clang driver on Windows doesn't support -stdlib=libc++
+// XFAIL: OS=windows-msvc
+
+// XFAIL: OS=linux-androideabi
+// XFAIL: OS=linux-android
+
+// CHECK-STD: import std.chrono
+// CHECK-STD: import std.iosfwd
+
+// CHECK-IOSFWD: extension std.__1 {
```

Comment:
This will probably need to be different on windows. I think it's `std.__cxx11.string` or something like that.